WcaOnRails/app/models/competition.rb

```diff
@@ -54,6 +55,17 @@ def must_have_at_least_one_delegate
     end
   end
 
+  # Currently we don't have a history of who was a delegate and when. Hence we need this
+  # validation, so people cannot pass a non-delegate as a delegate (even for an old comp).
+  # See https://github.com/cubing/worldcubeassociation.org/issues/185#issuecomment-168402252
+  # Once that is done, we might have to change this validation.
```

Comment:
Hmm... I wouldn't say this is a "might have to change this validation", it's more of a "we'll be able to change this validation to work on old competitions"